### ISSUE TITLE
ResNet as a front-end

### ISSUE THREAD

**Issue by atanas1054** [fyu/dilation#29]
Hi,

Do you think that the vgg16 can be replaced with ResNet? Would the idea of dilation work well?

**Comment by selcuksandikci** [comment]
Yes, it should work equally well. Already done in DeepLab segmentation paper in a slightly different way.

**Comment by nikhilparmar** [comment]
> Hi,
> 
> Do you think that the vgg16 can be replaced with ResNet? Would the idea of dilation work well?

Hello, can you guide me how have you replaced ResNet with VGG. I am thinking to replace the same in DeepLabv3+ for semantic segmentation.
